.env.default

```diff
@@ -31,16 +31,16 @@ DOCKERHUB_NAMESPACE=govcmslagoon
 X_FRAME_OPTIONS=SameOrigin
 
 # Set the PHP version to use for the upstream dockerfiles
-PHP_IMAGE_VERSION=7.3
+PHP_IMAGE_VERSION=7.4
 
 # Set the GovCMS version to use - you can use a tag or branch reference here
 # See https://github.com/govCMS/govcms/releases
-GOVCMS_PROJECT_VERSION=7.x-3.17
+GOVCMS_PROJECT_VERSION=7.x-3.21
 
 # Set the version of the site audit script to use - you can use a tag or branch reference here
 # See https://github.com/govCMS/audit-site/releases
 SITE_AUDIT_VERSION=7.x-3.x
 
-# Set the Lagoon tag to use for the upstream dockerfiles (e.g. v0.22.0) - make sure you change it for both lines
-# See https://github.com/amazeeio/lagoon/releases
-LAGOON_IMAGE_VERSION=v1.8.2
+# Set the Lagoon tag to use for the upstream dockerfiles (e.g. 20.12.0)
+# See https://hub.docker.com/r/uselagoon/php/tags
+LAGOON_IMAGE_VERSION=21.4.0
```

Comment:
Just noting that latest at this time is `21.6.0`

Reply:
Thanks Stuart for reviewing it for me.
We will consider update the Lagoon image to the latest version with next D7 release.